select.go

```diff
@@ -216,14 +219,16 @@ func (s *Select) RunCursorAt(cursorPos, scroll int) (int, string, error) {
 }
 
 func (s *Select) innerRun(cursorPos, scroll int, top rune) (int, string, error) {
-	stdin := readline.NewCancelableStdin(os.Stdin)
-	c := &readline.Config{}
+	c := &readline.Config{
```

Comment:
Does this change the API for select if these two are required? It should default to `os.Stdin` here (and `s.Stdout` should default to `os.Stdout`

Reply:
I don't think it does. `s.Stdin` and `s.Stdout` will be `nil` by default, and since they're passed as is to `readline.Config`, `c.Init()` will set `c.Stdin` to `NewCancelableStdin(os.Stdin)` and `c.Stdout` to `os.Stdout` for us. I reckon select could do this logic as well if we don't want to rely on the dependency for default behaviour, but the code as it is right now shouldn't break anything.